hw3_1.py

```diff
@@ -0,0 +1,8 @@
+res = list()
+array = [2, 2, 3, 3, 4]
+for el in array:
```

Comment:
Чтобы уменьшить итерации и убрать условие в 6, 7 строках
```suggestion
for el in set(array):
```